### PR TITLE
fix booking detail page for cancelled events

### PR DESCRIPTION
**Pull request by CarinaWolli**
## What does this PR do?

Fixes booking detail page for cancelled events.

Before: 
Screenshot 2022-05-27 at 14 52 49

After:
Screenshot 2022-05-27 at 14 44 51

**Environment**: Staging(main branch)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- Go to Bookings
- Click on a cancelled booking
- You should see the view from the screenshot above

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by agustif — APPROVED** [review]
Looks good to me! tested in local
image

**Review by alannnc — CHANGES_REQUESTED** [review]
image
I'm getting my attendees deleted @ api/cancel and then BookingListItem tries to display attendees[0] which don't exists and receive this error.
Probably we should reconsider not deleting attendees?

File: apps/web/components/booking/BookingListItem.tsx [hunk 1]
```diff
@@ -219,8 +219,8 @@ function BookingListItem(booking: BookingItemProps) {
               type: booking.eventType.id,
               eventSlug: booking.eventType.slug,
               user: user?.username || "",
-              name: booking.attendees[0].name,
-              email: booking.attendees[0].email,
+              name: booking.attendees[0] ? booking.attendees[0].name : undefined,
```
Comment (CarinaWolli) [inline apps/web/components/booking/BookingListItem.tsx:222]: this fixes the error from @alannnc's comment

File: apps/web/pages/success.tsx [hunk 2]
```diff
@@ -333,32 +343,36 @@ export default function Success(props: SuccessProps) {
...
-                                <p className="text-bookinglight">{bookingInfo.user.email}</p>
-                              </div>
-                            )}
-                            {bookingInfo?.attendees.map((attendee, index) => (
-                              <div
-                                key={attendee.name}
-                                className={index === bookingInfo.attendees.length - 1 ? "" : "mb-3"}>
-                                <p>{attendee.name}</p>
-                                <p className="text-bookinglight">{attendee.email}</p>
+                          {(bookingInfo?.user || bookingInfo?.attendees) && (
```
Comment (CarinaWolli) [inline apps/web/pages/success.tsx:351]: don't show 'who' when no user or attendees exist in booking

**Review by alannnc — APPROVED** [review]

**Pull request merged**